connector/connector.py

```diff
@@ -316,6 +327,29 @@ def get_connector_unit(self, base_class):
         return self.backend.get_class(base_class, self.session,
                                       self.model_name)(self)
 
+    @classmethod
+    def create_environment(cls, backend_record, session, model,
+                           connector_env=None):
+        """ Create a new environment ConnectorEnvironment.
+
+        :param backend_record: browse record of the backend
+        :type backend_record: :py:class:`openerp.models.Model`
+        :param session: current session (cr, uid, context)
+        :type session: :py:class:`connector.session.ConnectorSession`
+        :param model_name: name of the model
+        :type model_name: str
+        :param connector_env:
```

Comment:
Suggestion: an existing environment from which the kwargs will be propagated to the new one
Feel free to explain it in another way or improve my suggestion